src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -546,4 +557,4 @@ trait PluginFormats {
   ) (d => (d.id, d.plugin, d.implementation, d.tags, d.info))
 
   implicit lazy val pluginDefinitionsFormat: Writes[PluginDefinitions] = Json.writes[PluginDefinitions]
-}
+}
```

Comment:
It seems your editor is configured to drop trailing empty lines. Please make sure there are no such changes throughout this PR.

Reply:
yes，it no change，maybe my editor wrong。

Reply:
need i to re commit？

Reply:
Not re-commit, but push a new commit on top of the existing one in this branch.